model/model.go

```diff
@@ -78,6 +78,9 @@ func (c *ColumnInfo) Clone() *ColumnInfo {
 	return &nc
 }
 
+// ExtraHandleID is the column ID when we need to append an extra column as a handle.
```

Comment:
If pkIsHandle don't holds, we need a column to occupy the handle's position in the schema. And choose -1 is with no reason...